.github/workflows/dependabot_auto_merge.yaml

```diff
@@ -0,0 +1,22 @@
+name: Dependabot auto-merge
+
+on:
+  workflow_run:
+    types:
+      - completed
+    workflows:
+      - Continous Integration and Demo Deployment
```

Comment:
@nelsonkopliku this ensures that we run this workflow at the end of the CI workflow